### PR TITLE
MGDOBR-520 Support latest changes to Managed Connector API

### PR DESCRIPTION
**Pull request by lucamolteni**
* Upgrade to 0.19.0-beta1
* Use 0.15.5 for Kafka

This needs https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/573 to work on the dev environment, but at the same time the referred PR should be updated to effectively remove the cluster id as it's not used anymore and to update the kustomize yamls as well  @kostola 

[MGDOBR-520](https://issues.redhat.com/browse/MGDOBR-520)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [x] Check if code and Github action workflows have been modified and if the modification is safe
- [x] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by manstis — APPROVED** [review]
LGTM :+1:

**Comment by lucamolteni** [comment]
@sutaakar regarding https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/573#issuecomment-1088735747 I see that in the CI we use `MANAGED_CONNECTOR_CLUSTER_ID` while everywhere else we use `MC_CONNECTOR_CLUSTER_ID` we should probably uniform `MC` vs `MANAGED_CONNECTOR` WDYT?

File: manager/src/main/resources/application.properties [hunk 1]
```diff
@@ -102,7 +102,7 @@ quarkus.kafka.health.enabled=false
 mp.messaging.connector.smallrye-kafka.health-enabled=false
 
 # Managed Connectors
-managed-connectors.cluster.id=${MC_CLUSTER_ID:fakeClusterId}
+managed-connectors.namespace.id=${MC_CLUSTER_ID:fakeNamespaceId}
```
Comment (sutaakar) [inline manager/src/main/resources/application.properties:105]: Does it mean that namespace id == cluster id?So it is a same value, just renamed property?

**Comment by sutaakar** [comment]
@lucamolteni fine for me to uniform it

File: manager/src/main/resources/application.properties (hunk 1 above)
Comment (lucamolteni) [reply manager/src/main/resources/application.properties]: It was wrong, I fixed

**Comment by lucamolteni** [comment]
Updated values as for https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/573#issuecomment-1088735747 @robpblake @sutaakar please review again

**Comment by lucamolteni** [comment]
> @sutaakar regarding [#573 (comment)](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/573#issuecomment-1088735747) I see that in the CI we use `MANAGED_CONNECTOR_CLUSTER_ID` while everywhere else we use `MC_CONNECTOR_CLUSTER_ID` we should probably uniform `MC` vs `MANAGED_CONNECTOR` WDYT?

See https://issues.redhat.com/browse/MGDOBR-532

**Review by sutaakar — APPROVED** [review]

**Review by r00ta — APPROVED** [review]

**Review by robpblake — APPROVED** [review]

**Pull request merged**